bzip2-sys/wasm_shim.rs

```diff
@@ -0,0 +1,55 @@
+use std::alloc::{alloc, dealloc, Layout};
+use std::os::raw::{c_int, c_void};
+
+#[no_mangle]
+pub extern "C" fn rust_bzip2_wasm_shim_malloc(size: usize) -> *mut c_void {
+    unsafe {
+        let layout = Layout::from_size_align_unchecked(size, 1);
+        alloc(layout).cast()
+    }
+}
+
+#[no_mangle]
+pub extern "C" fn rust_bzip2_wasm_shim_calloc(nmemb: usize, size: usize) -> *mut c_void {
+    unsafe {
+        let layout = Layout::from_size_align_unchecked(size * nmemb, 1);
+        alloc(layout).cast()
```

Comment:
I don't think this is correct because I believe the memory needs to be zeroed to respect the contract of the `calloc` function.